sxw.c

```diff
@@ -1019,23 +1049,23 @@ void _print_debuginfo(void) {
 			//ForEachTranspLayer(t) {
 			fprintf(f, "%d", t + 1);
 			ForEachTrPeriod(p)
-				fprintf(f, "\t%.4f", _roots_active_rel[Iglp(r, t, p)]);
+				fprintf(f, "\t%.4f", SXWResources->_roots_active_rel[Iglp(r, t, p)]);
 			fprintf(f, "\n");
 		}
 	}
 
 	fprintf(f, "\n------ Transpiration Total Values -------\nPeriod:");
 	//ForEachTranspLayer(t)
-	for (t = 0; t < SXW.NSoLyrs; t++)
+	for (t = 0; t < SXW->NSoLyrs; t++)
 		fprintf(f, "\t\tL%d", t + 1);
 	fprintf(f, "\n");
 
 	ForEachTrPeriod(p)
 	{
 		fprintf(f, "%d : ", p + 1);
 		//ForEachTranspLayer(t)
```

Comment:
This line can probably safely be removed.

Reply:
Resolved by 103ca00.